### PR TITLE
Asgmt 580610624

### PR DESCRIPTION
**Pull request by Ngamsai**

### CODE REVIEW

File: app/src/main/res/layout/activity_option2.xml [hunk 1]
```diff
@@ -42,5 +42,23 @@ android:padding="@dimen/large_spacing">
...
     android:background="@color/colorAccent"/>
+    <RelativeLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+        <Button
+            android:id="@+id/back"
+            android:layout_width="173dp"
+            android:layout_height="wrap_content"
+            android:text="@string/back"
+            android:onClick="onClickBack" />
```
Comment (navadon) [inline app/src/main/res/layout/activity_option2.xml]: Do we still need this? (don't have to create a new pull request)

File: app/src/main/res/layout/activity_option2.xml [hunk 2]
```diff
@@ -42,5 +42,23 @@ android:padding="@dimen/large_spacing">
...
+
+        <Button
+            android:id="@+id/next"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:text="@string/next"
+            android:onClick="onClickNext"
```
Comment (navadon) [inline app/src/main/res/layout/activity_option2.xml]: Same issue as "onClickBack". Think about it. (no need to create a pull request)

**Comment by navadon** [comment]
Checked!

File: app/src/main/res/layout/activity_option2.xml (hunk 1 above)
Comment (Ngamsai) [reply app/src/main/res/layout/activity_option2.xml]: sorry , I forgot to delete these.

File: app/src/main/res/layout/activity_option2.xml (hunk 2 above)
Comment (Ngamsai) [reply app/src/main/res/layout/activity_option2.xml]: I have deleted it.

**Comment by Ngamsai** [comment]
I have editing already. Please check again.

**Pull request closed without merging**